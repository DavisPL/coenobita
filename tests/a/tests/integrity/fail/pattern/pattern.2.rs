#![allow(unused_variables)]
#![allow(unused_assignments)]

struct Baz {
    #[cnbt::tag({bin}{bin})]
    a: i64
}

fn main() {
    let x = Baz { a: 6 };

    let Baz { a } = x;

    #[cnbt::tag({bin}{bin})]
    let y = a;
}