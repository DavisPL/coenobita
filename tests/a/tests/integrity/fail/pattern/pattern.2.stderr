error: expected {bin}{bin}, found {bin}{*}
  --> tests/integrity/fail/pattern/pattern.2.rs:15:13
   |
15 |     let y = a;
   |             ^
